lib/reports/subscriber_lists_report.rb

```diff
@@ -0,0 +1,70 @@
+class Reports::SubscriberListsReport
+  attr_reader :start_of_day, :end_of_day
+
+  CSV_HEADERS = %i[title
+                   slug
+                   matching_criteria
+                   created_at
+                   individual_subscribers
+                   daily_subscribers
+                   weekly_subscribers
+                   individual_updates_for_date
```

Comment:
This could be confusing: it could refer to the number of emails sent (depends on subscription changes through the day), or the number of content changes and messages. Can we split it into (matched) content changes and messages?

Reply:
Sure thing 👍